salt/metalk8s/kubernetes/apiserver/pre-upgrade.sls

```diff
@@ -0,0 +1,11 @@
+{%- set cp_nodes = salt.metalk8s.minions_by_role('master') | sort %}
+
+# Always run this state as it do not change anything if cert are already good
```

Comment:
Could read:
`Always run this state as it does not change anything if certs are already good`